pom.xml

```diff
@@ -66,90 +64,73 @@
         <logback.version>1.2.3</logback.version>
         <ipprefix>127.0.1.</ipprefix>
+        <!-- defaults below are overridden by profiles and/or submodules -->
         <test.groups>unit</test.groups>
-        <!-- Set default javadoc.opts, overriden by profiles -->
+        <test.osgi.skip>true</test.osgi.skip>
         <javadoc.opts />
     </properties>
 
     <profiles>
-        <profile>
-            <id>default</id>
-            <properties>
-                <env>default</env>
-                <test.groups>unit</test.groups>
-            </properties>
-            <activation>
-                <activeByDefault>true</activeByDefault>
-            </activation>
-        </profile>
-
-        <profile>
-            <id>doclint-java8-disable</id>
-            <activation>
-                <jdk>[1.8,)</jdk>
-            </activation>
-            <properties>
-                <javadoc.opts>-Xdoclint:none</javadoc.opts>
-            </properties>
-        </profile>
 
         <profile>
             <id>short</id>
             <properties>
-                <env>default</env>
                 <test.groups>unit,short</test.groups>
+                <test.osgi.skip>false</test.osgi.skip>
             </properties>
         </profile>
+
         <profile>
             <id>long</id>
             <properties>
-                <env>default</env>
                 <test.groups>unit,short,long</test.groups>
+                <test.osgi.skip>false</test.osgi.skip>
             </properties>
         </profile>
+
         <profile>
             <id>duration</id>
             <properties>
-                <env>default</env>
                 <test.groups>unit,short,long,duration</test.groups>
+                <test.osgi.skip>false</test.osgi.skip>
             </properties>
         </profile>
+
         <profile>
             <id>doc</id>
             <properties>
-                <env>default</env>
                 <test.groups>unit,doc</test.groups>
             </properties>
         </profile>
+
+        <!-- default profile settings for 'isolated' test group, will skip tests unless overridden in child module. -->
         <profile>
-            <!-- default profile settings for 'isolated' test group, will skip tests unless overridden in child module. -->
             <id>isolated</id>
             <properties>
-                <env>default</env>
                 <test.groups>isolated</test.groups>
             </properties>
             <build>
                 <plugins>
                     <plugin>
-                        <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-surefire-plugin</artifactId>
-                        <version>2.16</version>
+                        <!-- do not upgrade until https://issues.apache.org/jira/browse/SUREFIRE-1302 is fixed -->
+                        <version>2.18</version>
                         <configuration>
-                            <properties>
```

Comment:
This was done for #730, is it no longer needed with 2.18 or is there some other change that makes it no longer needed?